### DISCUSSION TITLE
Apps billing strategy and push notifications

### DISCUSSION THREAD

**Discussion by jericopulvera** [evoluhq/evolu#299] [Q&A] [answered]
To ensure clarity, it’s important to store a distinct identifier on the backend for managing access to online services. Given the local-first nature of the app, preventing users from using it is impractical. I plan to offer a few paid online services, such as push notifications and a sync server, with each user associated with a unique identifier to regulate access.

What are the best way to achieve this? Do I store their mnemonic on the online services backend?

**Comment by steida** [comment, marked as answer]
No, just store which OwnerId/UserId is paid.

**Comment by steida** [comment]
This will be improved soon
